README.md

```diff
@@ -29,10 +36,19 @@ android_library(
 )
 ```
 
-You can see the full list of generated targets in [gmaven.bzl](https://raw.githubusercontent.com/aj-michael/gmaven_rules/master/gmaven.bzl).
+You can see the full list of generated targets in [`gmaven.bzl`](https://raw.githubusercontent.com/aj-michael/gmaven_rules/master/gmaven.bzl).
 
-To regenerate gmaven.bzl, run the following command. It will take about 5 minutes.
+# Regenerating targets
```

Comment:
nit: Perhaps change the title to "Regenerating gmaven.bzl" or "Updating gmaven.bzl"? This doesn't actually generate targets, it generates repository rules

Reply:
done